jupyter_server/services/contents/handlers.py

```diff
@@ -424,6 +424,14 @@ async def post(self, path=""):
         self.finish()
 
 
+# Somehow this doesn't feel like the right service for this to go in?
```

Comment:
Let's move this into a new folder + service.

Let's do `jupyter_server/services/tools` as the parent folder and add this handler under `jupyter_server/services/tools/handlers.py`.